src/auto-evo/selection_pressure/MetabolicStabilityPressure.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace AutoEvo;
+
+public class MetabolicStabilityPressure : SelectionPressure
+{
+    // Needed for translation extraction
+    // ReSharper disable ArrangeObjectCreationWhenTypeEvident
+    private static readonly LocalizedString NameString = new LocalizedString("MICHE_METABOLIC_STABILITY_PRESSURE");
+
+    // ReSharper restore ArrangeObjectCreationWhenTypeEvident
+    private static readonly Compound ATP = SimulationParameters.Instance.GetCompound("atp");
```

Comment:
I'll stop checking these pressures for now as many seem to have the same static variable storing registry data problem.